executor/show.go

```diff
@@ -1331,7 +1331,7 @@ func (e *ShowExec) fetchShowCreateUser() error {
 
 	exec := e.ctx.(sqlexec.RestrictedSQLExecutor)
 
-	stmt, err := exec.ParseWithParams(context.TODO(), `SELECT * FROM %n.%n WHERE User=%? AND Host=%?`, mysql.SystemDB, mysql.UserTable, userName, hostName)
+	stmt, err := exec.ParseWithParams(context.TODO(), `SELECT plugin FROM %n.%n WHERE User=%? AND Host=%?`, mysql.SystemDB, mysql.UserTable, userName, hostName)
```

Comment:
I think this should still be `SELECT *`?

Reply:
I don't think that's the case. Previously only `len(rows)` was checked (line 1338) before rows being overwritten again with the next statement on line 1353. Now only the `plugin` field is used with `rows[0].GetString(0)` if there is 1 row.

I could change this back to an `SELECT *` and then use `GetStringByName` but I think it is better to explicitly name the columns that are needed.

Reply:
Makes sense, thanks! It just needs a query to test existence - but you've changed this to get the plugin name.